support-workers/src/typescript/model/stateSchemas.ts

```diff
@@ -0,0 +1,159 @@
+import { z } from 'zod';
+import { addressSchema } from './address';
+import { paymentFieldsSchema, paymentProviderSchema } from './paymentFields';
+import { paymentMethodSchema } from './paymentMethod';
+import { productTypeSchema } from './productType';
+
+export const titleSchema = z.union([
+	z.literal('Mr'),
+	z.literal('Mrs'),
+	z.literal('Ms'),
+	z.literal('Miss'),
+	z.literal('Mx'),
+	z.literal('Dr'),
+	z.literal('Prof'),
+	z.literal('Rev'),
+]);
+
+export const userSchema = z.object({
+	id: z.string(),
+	primaryEmailAddress: z.string(),
+	title: titleSchema.nullable(),
+	firstName: z.string(),
+	lastName: z.string(),
+	billingAddress: addressSchema,
+	deliveryAddress: addressSchema.nullable(),
+	telephoneNumber: z.string().nullable(),
+	isTestUser: z.boolean().default(false),
```

Comment:
I think it's always in the json
```suggestion
	isTestUser: z.boolean(),
```
the original case class is not a json deserialiser, it's just a scala model.
This line would only make it false when constructed from scala code
https://github.com/guardian/support-frontend/blob/f2bbb0a87cf906276a7a17e12a36a5e5a0d1a0c6/support-models/src/main/scala/com/gu/support/workers/User.scala#L16
whereas the line written above would return false if it's not set in the json
https://zod.dev/?id=default